checkov/secrets/runner.py

```diff
@@ -177,13 +177,14 @@ def run(
                     severity=severity,
                     check_name=secret.type,
                     check_result=result,
-                    code_block=[(secret.line_number, line_text_censored, len(cast(str, secret.secret_value)))],
+                    code_block=[(secret.line_number, line_text_censored)],
                     file_path=f'/{os.path.relpath(secret.filename, root_folder)}',
                     file_line_range=[secret.line_number, secret.line_number + 1],
                     resource=secret.secret_hash,
                     check_class="",
                     evaluations=None,
                     file_abs_path=os.path.abspath(secret.filename),
+                    check_len=len(cast(str, secret.secret_value)),
```

Comment:
can you create a new test or adjust an existing one to check for this value?

Reply:
I'd highly suggest avoiding any change to the `Record` model unless absolutely necessary

Reply:
because we don't pass the full check i have to do it here. 
my first thought was to filter the checks here but i need to filter them only if it is a custom policy so i need to do it in the platform. will add UT